WinUIGallery/ItemPage.xaml.cs

```diff
@@ -33,6 +33,9 @@ namespace AppUIBasics
     /// </summary>
     public sealed partial class ItemPage : Page
     {
+        private static string GalleryBaseUrl = "https://github.com/microsoft/WinUI-Gallery/tree/main/WinUIGallery/ControlPages/";
+        private static string WinUIBaseUrl = "https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.4-stable/controls/dev/";
```

Comment:
```suggestion
        private static string WinUIBaseUrl = "https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.4-stable/controls/dev/";
        private static string GalleryBaseUrl = "https://github.com/microsoft/WinUI-Gallery/tree/main/WinUIGallery/ControlPages/";
```
Just to follow the UI order 😇